catatonit.c

```diff
@@ -102,7 +102,7 @@ static void help(void)
 
 static void version(void)
 {
-	fprintf(stdout, "tini version %s_%s\n", PROGRAM_VERSION, PROGRAM_NAME);
+	fprintf(stdout, "%s version %s_%s\n", PROGRAM_NAME, PROGRAM_VERSION, PROGRAM_NAME);
```

Comment:
This was intentional to make `docker-info` happy. Docker is hard-coded against tini and this was the (unfortunate) hack to make it work nicely with catatonit.

Did Docker's behavior change?

Reply:
Well, I have no idea. I thought this was just a silly typo :smile:. If this is not the case, maybe I can add a comment explaining just that.